### PR TITLE
added fast algo

### PR DESCRIPTION
**Pull request by jlovescode**
The algo is much faster

### CODE REVIEW

File: Algorithms/src/main/java/com/godwije/algos/FastSearch.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.godwije.algos;
+
+import java.util.HashSet;
+import java.util.Set;
+
```
Comment (jeswingodwin) [inline Algorithms/src/main/java/com/godwije/algos/FastSearch.java:5]: Can you please add more comments for your search algo

**Review by jeswingodwin — CHANGES_REQUESTED** [review]
Add comments as shown

File: Algorithms/src/main/java/com/godwije/algos/FastSearch.java (hunk 1 above)
Comment (jlovescode) [reply Algorithms/src/main/java/com/godwije/algos/FastSearch.java:5]: Okay Sure

**Review by jlovescode — COMMENTED** [review]
okay I will change

**Review by jeswingodwin — APPROVED** [review]

**Pull request merged**